from_mwt_ds/DataScience/estimate.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+
+#  Way to workaround while it is not in pip
+#  1. clone estimators/rlos2021_cfe, python3 setup.py install
+#  2. copy estimators folder from rlos2021_cfe branch of estimators repo
+# import estimators 
+
+def estimate_bucket(bucket: pd.DataFrame, config: dict) -> dict: 
+    if len(bucket) == 0 or config == {}:
+        return {}
+    else:
+        return_dict = {}
+        for policies in config:
+            firstpart = list(config.get(policies).keys()[0])
+            for i in config.get(policies):
+                secondpart = list(config.get(policies).values())[0][0]
+        keyresult = firstpart + "_" + secondpart
+        formula_ips = (df["r"][0] * df["policy_1"][0] / df["p"][0]) / 1
```

Comment:
Looks like you are reimplementing ips here.
Can we use reference to estimator library here?